extract_endpoint/src/azure_utils.py

```diff
@@ -0,0 +1,38 @@
+import enum
+from io import BytesIO
+from typing import Optional, NamedTuple
+from azure.storage.blob import BlockBlobService
+
+
+class StorageCoordinates(NamedTuple):
+    account: str
+    key: str
+    container: str
+    domain: Optional[str]
+
+
+class EnvVariables(enum.Enum):
+    account = 'EXTRACT_ENDPOINT_STORAGE_ACCOUNT'
+    key = 'EXTRACT_ENDPOINT_STORAGE_KEY'
+    container = 'EXTRACT_ENDPOINT_CONTAINER'
+    domain = 'EXTRACT_ENDPOINT_STORAGE_DOMAIN'
+
+
+class DefaultVariables(enum.Enum):
+    account = 'devstoreaccount1'
+    key = 'Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=='
```

Comment:
Is this a real key to Azure blob storage?

Reply:
I believe (but Steve should verify), that that is the key to the local emulator.

Reply:
yes, local emulator

Reply:
I'm confused. Is that the key to everyone's local emulator? Does it ever change?

Reply:
The emulator is preconfigured to use this for the key (along with the account name `devstoreaccount1`)

Reply:
> The emulator has only one built in account that you cannot edit, the account details are as follows:
Account name: devstoreaccount1
Account key: 
Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==

http://blog.podrezo.com/azure-storage-emulator/
👍